sechub-pds-solutions/findsecuritybugs/env-cluster-initial

```diff
@@ -19,4 +19,5 @@ S3_ACCESSKEY=user
 S3_SECRETKEY=Why$5forThat?
 PDS_MAX_FILE_UPLOAD_SIZE=-1
 PDS_UPLOAD_MAXIMUM_BYTES=262144000
-PDS_VERSION=0.31.0
\ No newline at end of file
+PDS_VERSION=0.31.0
+BASE_IMAGE="ghcr.io/mercedes-benz/sechub/pds-base:v0.32.1"
```

Comment:
Do we want to have two different versions here (`0.31.0` and `0.32.1`)? Unless it's two different things

Reply:
We've done a quick lookup regarding the version differences, Dockerfiles at quite frequently set `ARG PDS_VERSION="0.31.0"` and version of `BASE_IMAGE` is usually `0.32.1`.  

I checked if it would work with another version of `BASE_IMAGE` but it behaves like there's no suitable package present, so i guess `BASE_IMAGE` needs to stay as is.  

`PDS_VERSION="0.32.1" works on the other hand.

I would create PR to upstream and comment there that we've seen version differences and whether we should to something about it.